src/_TextInputInnerAction.tsx

```diff
@@ -77,7 +82,7 @@ const TextInputAction = forwardRef<HTMLButtonElement, TextInputActionProps>(
     }
 
     return (
-      <Box as="span" className="TextInput-action" margin={1}>
+      <Box as="span" className="TextInput-action" marginLeft={1} marginRight={1}>
```

Comment:
This overall height was a bit off here, but I noticed `small` and `large` look too large with a trailing action as well. We could move this to another PR and address that separately as well!

Reply:
This seems like a small enough change to sneak into this PR 👍